diff --git a/node_modules/react-native-postcss-transformer/index.js b/node_modules/react-native-postcss-transformer/index.js
index 0f06196..bc42ed4 100644
--- a/node_modules/react-native-postcss-transformer/index.js
+++ b/node_modules/react-native-postcss-transformer/index.js
@@ -8,7 +8,9 @@ var upstreamTransformer = null;
 var reactNativeVersionString = require("react-native/package.json").version;
 var reactNativeMinorVersion = semver(reactNativeVersionString);
 
-if (reactNativeMinorVersion >= 59) {
+if (reactNativeMinorVersion >= 73) {
+  upstreamTransformer = require("@react-native/metro-babel-transformer");
+} else if (reactNativeMinorVersion >= 59) {
   upstreamTransformer = require("metro-react-native-babel-transformer");
 } else if (reactNativeMinorVersion >= 56) {
   upstreamTransformer = require("metro/src/reactNativeTransformer");
